README.md

```diff
@@ -1,143 +1,105 @@
 # The CESE Hub
 This repository is for [The CESE Hub](http://cesehub.org/). The CESE Hub is a web site to introduce and collect information for [the space-time conservation element and solution element (CESE) method](http://www.grc.nasa.gov/WWW/microbus/).
-# The CESE Hub
-This repository is for [The CESE Hub](http://cesehub.org/). The CESE Hub is a web site to introduce and collect information for [the space-time conservation element and solution element (CESE) method](http://www.grc.nasa.gov/WWW/microbus/).
 
-## Welcome to contribute codes to this website!!
```

Comment:
I think it's not needed to use the exclamation marks.

Because in line 1 the section title is capitalized, we should be consistent and capitalize all titles.